bigquery/biglake/bigquery_create_unpartitioned_biglake_table/main.tf

```diff
@@ -81,7 +81,13 @@ resource "google_bigquery_dataset" "default" {
 
 
 # This creates a BigQuery table named "my_table" in the dataset "default".
-# The table has three columns, named "country", "product", and "price", which are of types STRING, STRING, and INT64 respectively.
+# The table has three columns, named "country", "product", and "price", which
+# are of types STRING, STRING, and INT64 respectively.
+# The table has Automatic Metadata Caching enabled.
+# https://cloud.google.com/bigquery/docs/biglake-intro#metadata_caching_for_performance
+# `metadata_cache_mode` may be set to `MANUAL` and `max_staleness` ommitted to
+# use MANUAL Metadata refresh `metadata_cache_mode` and `max_staleness` may be
+# set to omitted to disable the feature.
 resource "google_bigquery_table" "default" {
   depends_on = [time_sleep.wait_7_min]
```

Comment:
Move `depends_on` argument to the last. 

[Style Guide](https://developer.hashicorp.com/terraform/language/syntax/style) says `For blocks that contain both arguments and "meta-arguments" (as defined by the Terraform language semantics), list meta-arguments first and separate them from other arguments with one blank line. Place meta-argument blocks last and separate them from other blocks with one blank line.`

Reply:
done.

copied to all blocks and files.